app/src/examples/VolumeExample.tsx

```diff
@@ -0,0 +1,211 @@
+          animatedProps={animatedTextProps}
+        />
+      </View>
+      <Animated.View
+        style={[styles.row, styles.center, styles.margin, animatedStyle]}>
+        <Text style={styles.text}>0</Text>
+        <AnimatedSlider
+          style={styles.slider}
+          minimumValue={0}
+          maximumValue={100}
+          minimumTrackTintColor={primaryColor}
+          maximumTrackTintColor={secondaryColor}
+          thumbImage={require('./assets/doge.png')}
+          animatedProps={animatedProps}
+        />
+        <Text style={styles.text}>100</Text>
+      </Animated.View>
+      <View style={[styles.row, styles.between, styles.margin]}>
+        <Text style={styles.text}>Change with buttons</Text>
+        <Switch
+          trackColor={{ false: secondaryBackgroundColor, true: primaryColor }}
+          thumbColor={secondaryColor}
+          ios_backgroundColor={secondaryBackgroundColor}
+          value={true}
+        />
+      </View>
+      <View style={styles.divider} />
+      <Text style={[styles.header, styles.margin]}>Streaming quality</Text>
+      <Text style={styles.text}>Automatic</Text>
+      <Text style={styles.text}>Low</Text>
+      <Text style={styles.text}>Normal</Text>
+      <Text style={styles.text}>High</Text>
+      <Text style={styles.text}>Very high</Text>
+      <View style={styles.margin} />
+      <View style={[styles.row, styles.between, styles.margin]}>
+        <Text style={styles.text}>Adjust audio quality</Text>
+        <Switch
+          trackColor={{ false: secondaryBackgroundColor, true: primaryColor }}
+          thumbColor={secondaryColor}
+          ios_backgroundColor={secondaryBackgroundColor}
+          value={true}
+        />
+      </View>
+      <View style={styles.divider} />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  text: {
+    color: secondaryColor,
+    fontSize: 16,
+    padding: 4,
+    fontFamily: 'Poppins-Regular',
+  },
+  row: {
+    flexDirection: 'row',
+  },
+  margin: {
+    marginBottom: 16,
+  },
+  between: {
+    justifyContent: 'space-between',
+  },
+  center: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  slider: {
+    height: 40,
+    alignSelf: 'center',
+    flex: 1,
+  },
+  valueContainer: {
+    backgroundColor: secondaryColor,
+    borderRadius: 16,
+    borderWidth: 1,
+    width: 170,
+    height: 120,
+    alignSelf: 'center',
+    justifyContent: 'center',
+  },
+  value: {
+    textAlign: 'center',
+    // fontFamily: 'Poppins-Bold',
+    fontWeight: 'bold',
+    fontVariant: ['tabular-nums'],
+    fontSize: 70,
+    color: backgroundColor,
+  },
+  divider: {
+    height: StyleSheet.hairlineWidth,
+    backgroundColor: secondaryColor,
+    marginTop: 16,
+    marginBottom: 32,
+  },
+  header: {
+    fontSize: 24,
+    color: secondaryColor,
+    fontFamily: 'Poppins-Bold',
+  },
+  subHeader: {
+    fontSize: 24,
+    color: secondaryColor,
+    fontFamily: 'Poppins-Bold',
```

Comment:
Do you think if it makes sense to have `const FONT_FAMILY = 'Poppins-Bold';` and use `FONT_FAMILY` everywhere instead?

Reply:
Valid point but there are only two `Poppins-Bold` and one `Poppins-Regular`. If there would be more then I'd definitely do that but I think here it's fine to leave it as is